dev/io.openliberty.microprofile.telemetry.common/resources/io/openliberty/microprofile/telemetry/common/resources/MPTelemetry.nlsprops

```diff
@@ -37,4 +37,9 @@ CWMOT5000.cannot.get.globalopentelemetry.useraction=Use CDI to inject the OpenTe
 
 CWMOT5001.cannot.set.globalopentelemetry=CWMOT5001E: Setting the GlobalOpenTelemetry object is not supported.
 CWMOT5001.cannot.set.globalopentelemetry.explanation=The GlobalOpenTelemetry class is not supported because no global OpenTelemetry object exists. Instead, each application must use CDI to inject the OpenTelemetry object for that application.
-CWMOT5001.cannot.set.globalopentelemetry.useraction=Do not call the GlobalOpenTelemetry.get method. Where the application code needs to obtain an OpenTelemetry object, use CDI to inject it.
\ No newline at end of file
+CWMOT5001.cannot.set.globalopentelemetry.useraction=Do not call the GlobalOpenTelemetry.get method. Where the application code needs to obtain an OpenTelemetry object, use CDI to inject it.
+
+CEMOT5102.telemetry.error=CEMOT5102: An internal error occurred in MicroProfile Telemetry. The error is: {0}.
+CEMOT5102.telemetry.error.explanation=An error has occurred that prevents normal operation of MicroProfile Telemetry.
+CEMOT5102.telemetry.error.useraction=Review the server message.log file and FFDC logs to identify the problem.
```

Comment:
```suggestion
CWMOT5002.telemetry.error=CWMOT5002E: An internal error occurred in MicroProfile Telemetry. The error is: {0}.
CWMOT5002.telemetry.error.explanation=An error has occurred that prevents normal operation of MicroProfile Telemetry.
CWMOT5002.telemetry.error.useraction=Review the server message.log file and FFDC logs to identify the problem.
```

`CWMOT` is the message prefix block. We would put the `E` to signify an error on the end of the message ID.

In addition, the next free ID is 5002. (We could also use 5101 since we have a 5100, but I think extending the 500x series makes more sense.)

References to the message ID unfortunately need to be updated throughout.